docs/src/main/asciidoc/infinispan-client.adoc

```diff
@@ -357,6 +424,27 @@ annotation is not required and if it is not supplied, the default cache will be
 
 NOTE: Other types may be supported for injection, please see other sections for more information
 
+=== Registering Protobuf Schemas with Infinispan Server
+
+One the serialization on the client guide is done, you must register Protobuf schemas
+with Infinispan Server if you want to perform queries or convert from `Protobuf` to other media types
+such as `json`.
+
+[TIP]
+If you log to the Infinispan Console for a local running server in `http://localhost:11222`,
+you can check the schemas that exist under the `Schemas` tab.
+
+The `quarkus.infinispan-client.use-schema-registration` property is `true` by default, so the extension
+will take care of the schema upload to Infinispan. This is very useful for development mode.
+However, if you need to deal yourself with the schema evolution over time,
+turn the property to 'false' for production environments.
```

Comment:
```suggestion
By default protobuf schemas generated this way will be registered by this extension
when the client first connects.
However, it might be required to handle the registration manually as a schema may evolve
over time when used in production, so you can disable this from occurring by configuring the 
`quarkus.infinispan-client.use-schema-registration` to `false`.
```